src/Orleans/Streams/Predicates/ExactMatchStreamNamespacePredicate.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Orleans.Streams
+{
+    [Serializable]
+    internal class ExactMatchStreamNamespacePredicate : IStreamNamespacePredicate
```

Comment:
@xiazen It is used in `ImplicitStreamSubscriptionAttribute`, so I didn't see a point to expose it externally, since the users can just use `ImplicitStreamSubscriptionAttribute` with ctor accepting the stream namespace.